nautobot/core/views/generic.py

```diff
@@ -1354,7 +1344,8 @@ def post(self, request):
                         elif field == "label":
                             field = "label_pattern"
                         for e in errors:
-                            form.add_error(field, "{}: {}".format(name, ", ".join(e)))
+                            e = ", ".join(e)
+                            form.add_error(field, f"{name}: {e}")
```

Comment:
As we re-enable more pylint checkers, pylint probably won't be happy about the redefinition of `e`. Perhaps:

```suggestion
                            err_str = ", ".join(e)
                            form.add_error(field, f"{name}: {err_str}")
```